src/fides/api/alembic/migrations/versions/1e750936a141_migrate_to_fideslang_2_0_uses_and_.py

```diff
@@ -0,0 +1,127 @@
+"""migrate to fideslang 2.0 uses and categories
+
+Revision ID: 1e750936a141
+Revises: fd52d5f08c17
+Create Date: 2023-08-17 08:51:47.226612
+
+"""
+import json
+from typing import Dict, List, Optional
+
+from alembic import op
+from loguru import logger
+from sqlalchemy import text
+from sqlalchemy.engine import Connection, ResultProxy
+from sqlalchemy.sql.elements import TextClause
+
+# revision identifiers, used by Alembic.
+revision = "1e750936a141"
+down_revision = "708a780b01ba"
+branch_labels = None
+depends_on = None
+
+###############
+## Data Uses ##
+###############
+# The `key` is the old value, the `value` is the new value
+data_use_upgrades: Dict[str, str] = {
+    "improve.system": "functional.service.improve",  # Verified in 2.0
```

Comment:
what if a customer had branched their taxonomy off of "improve.system," and they now have "improve.system.maintenance" and added that to a privacy declaration.  We didn't handle this sort of thing last time, is this just handled on a customer by customer basis?

Reply:
OK learning more, since we don't remove the old uses from the db, it's fine if their privacy declarations continue to reference their custom uses that are based off of data uses we've technically deprecated.  I assume.

The custom taxonomies are an interesting use case.

Reply:
Correct, these data migrations only care about "renames", not removals/additions